modules/nf-core/abra2/realign/main.nf

```diff
@@ -0,0 +1,60 @@
+
+process ABRA2_REALIGN {
+    tag "$meta.id" + (binding.variables["meta2"] ? ",$meta2.id" : ",none") + (binding.variables["meta3.id"] ? ",$meta3.id" : ",none")
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::abra2=2.24" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/quay.io/biocontainers/abra2:2.24--h9f5acd7_1':
+        'quay.io/biocontainers/abra2:2.24--h9f5acd7_1' }"
+
+    input:
+    tuple val(meta), path(bam),path(bam_bai)
+    tuple val(meta2), path(bam2),path(bam_bai2)
+    tuple val(meta3), path(bam3),path(bam_bai3)
+    tuple path(fasta), path(fasta_fai)
```

Comment:
```suggestion
    path fasta
    path fasta_fai
```

Suggestion to be along the same lines as other modules that use `fasta` and `fasta_fai`